engine/src/lib.rs

```diff
@@ -39,6 +41,8 @@ pub struct Engine {
     network_inbox: Vec<MessageData>,
     /// Configuration we were constructed with
     cfg: Config,
+    /// Timing
```

Comment:
Maybe just use the comment that Timing has lol

Reply:
Noted